### DISCUSSION TITLE
Bootstrap 5 final release date

### DISCUSSION THREAD

**Discussion by gboscolo** [twbs/bootstrap#33617] [Q&A] [answered]
Hi there,
we are starting a new web project and we're facing the problem to choose which version of Bootstrap to use: version 5 is still in beta and there's not a certain date of the final stable release. On the other side, if we use the 4.x version, we would avoid to upgrade this to the 5 after less than 6 months because it goes in a declared end-of-life on january 2022.
So, is there a way to know a release date for the 5 stable version? 
Thanks,
Gianluca

**Comment by mdo** [comment, marked as answer]
Hoping for a stable release in the next couple weeks. I've updated https://github.com/twbs/release to push back the LTS and EOL for v4 by another six months. We won't have final dates on those v4 milestones until v5 lands.

**Reply by gboscolo** [reply]
So, talking by dev to dev, do you suggest to use the 5 beta version in production projects to be released on july 21? Is it sure that on this date a stable version will be available?
Another question, using Edge Chromium with IE11 mode on will be supported by Bootstrap 5? It is a common case use of our customers so we need to be sure about that
Many thanks!

**Reply by wnunezc** [reply]
version 5 is already a couple of months late, it is supposed to be relase in February.
https://github.com/twbs/release#bootstrap-5

I think that beta 3 has few problems that negatively not impact any production scenario and the changes from the official version 5.0 should be minimal.

I am also anxious and impatient for this release ... I really have several projects in hand that will implement BS5 ... I go in every 15 minutes to see if there is something new!!!

**Reply by gboscolo** [reply]
Yeah! Do you know if there's a official forum where to ask for the date?

**Reply by mdo** [reply]
This is it, and hoping for in the next week or two!

**Reply by gboscolo** [reply]
That's great! many thanks! ;)

**Reply by wnunezc** [reply]
no

**Reply by lorvent** [reply]
If i were you, i would start with BS5 because it is already in beta and there won't be much breaking changes anyways.
because if you start with bs4 now, once bs5 released....you will definitely need to upgrade to bs5 because client don't want bs4 in July as bs5 is already released by then.

and looking at projects tab, you can see bs5 is about to be release....just final touches are pending.

**Reply by gboscolo** [reply]
@mdo  what about Edge in ie11 mode? will it work? Thanks

**Comment by coliff** [comment]
@gboscolo IE Mode in Edge uses the IE11 rendering engine (https://docs.microsoft.com/en-us/deployedge/edge-ie-mode) so Bootstrap 5 won't render properly. You'd need to use polyfills - https://github.com/coliff/bootstrap-ie11
